docs/deployment/image-variants.md

```diff
@@ -0,0 +1,38 @@
+---
+title: "Image variants"
+layout: default
+sort: 1
+---
+
+# Image variants
+{: .no_toc}
+
+## Table of contents
+{: .no_toc .text-delta}
+
+1. TOC
+{:toc}
+
+---
+
+NFD-Operator currently offers two variants of the container image. The "minimal" variant is
+currently deployed by default. Released container images are available for
+x86_64 and Arm64 architectures.
+
+## Minimal
+
+This is a minimal image based on
+[gcr.io/distroless/base](https://github.com/GoogleContainerTools/distroless/blob/master/base/README.md)
+and only supports running statically linked binaries.
+
+For backwards compatibility a container image tag with suffix `-minimal`
+(e.g. `{{ site.container_image }}-minimal`) is provided.
```

Comment:
Maybe adapt this description to the operator. Statically linked binaries is not relevant, I think